subprojects/docs/src/samples/userguideOutput/nativeAssembleDependentComponents.out

```diff
@@ -1,14 +1,18 @@
+:discoverInputsForCompileOperatorsTestPassingCUnitExeOperatorsC
```

Comment:
Being shorter doesn't seem to be the goal of the software model task names 😉 
Anyway, should we go for `discoverInputsForOperatorsTestPassingCUnitExeOperatorsC` or `discoverInputsOperatorsTestPassingCUnitExeOperatorsC`? The second version would be even shorter and be more in line with `compileOperatorsTestPassingCUnitExeOperatorsC` and `linkOperatorsTestPassingCUnitExeOperatorsC`. I am also happy to use something different to `discoverInputs`. We could go for something like `detectUsedHeaders` or `findHeaders`. Something along `detectDependencies` could also work, since that is what gcc calls this output, right (See [preprocessor options](https://gcc.gnu.org/onlinedocs/gcc-4.4.1/gcc/Preprocessor-Options.html) -> `-M`).

Reply:
I would call it `depend` + something.  That's what most native people would be familiar with (`.d` files are "depend" files).

Reply:
I like `detectHeaders` the best. Maybe `depend` could work, too, but the verb does not actually describe what the task does to its inputs. I don't have strong feelings whether or not to include `For` in the name.

Reply:
I changed the name to `dependXXX`, `XXX` does not contain `compile`.